koku/masu/processor/aws/aws_report_db_cleaner.py

```diff
@@ -60,8 +60,7 @@ def purge_expired_line_item(self, expired_date, provider_uuid=None, simulate=Fal
             err = "Parameter expired_date must be a datetime.datetime object."
             raise AWSReportDBCleanerError(err)
 
-        with ReportingCommonDBAccessor() as reporting_common:
-            column_map = reporting_common.column_map
+        column_map = REPORT_COLUMN_MAP
```

Comment:
Would it be possible to move the column_map definition up into the `__init__` function for the class like how you did it here: https://github.com/project-koku/koku/pull/2035/files#diff-b9445f144f35a736ed64f2df640a163cR48

So like:
```
self._column_map = REPORT_COLUMN_MAP
```

Reply:
@nbon12 Can you respond to this comment?

Reply:
@kholdaway Cody and I discussed that it's not necessary at this time.

Reply:
Can you provide what he said on the PR?

Reply:
“Totally I will resolve it.”

Reply:
I will follow up with Cody.  I am not sure why it's not related to your code.